prometheus_client/multiprocess.py

```diff
@@ -82,21 +82,22 @@ def _parse_key(key):
     def _accumulate_metrics(metrics, accumulate):
         for metric in metrics.values():
             samples = defaultdict(float)
-            buckets = {}
+            buckets = defaultdict(lambda: defaultdict(float))
+            samples_setdefault = samples.setdefault
             for s in metric.samples:
-                name, labels, value = s.name, s.labels, s.value
+                name, labels, value, timestamp, exemplar = s
```

Comment:
timestamp and exemplar aren't used

Reply:
True. This is another micro-optimization: unpacking a slice of `s` would be slower than unpacking everything.